Automattic/sensei
# Remove WooThemes prefix from widgets

I thought this was going to be harder because widgets are registered with their class names. Turns out, widgets that extend `WP_Widget` use the ID passed to the parent constructor for all unique naming. This means that previous installs shouldn't lost their Sensei placed widgets. Additionally, the hooks used in our widgets use a predefined prefix that doesn't contain `woothemes`. 🎉 

### Testing Instructions
- From `master`, add all of Sensei core's widgets to sidebar areas.
- Switch to this branch. Load the front end and make sure they are all still there.
- Make sure no errors are logged/thrown.
